src/app/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,12 @@
+{% autoescape off %}
+To initiate the password reset process for your {{ user.get_username }} SPDX Online tools Account,
+click the link below:
+
```

Comment:
Username : {{ user.get_username }}
To initiate the password reset process for your SPDX Online tools Account, click the link below